Android/app/src/main/java/io/github/project_travel_mate/destinations/description/PlacesOnMap.java

```diff
@@ -195,7 +213,7 @@ public void onMapReady(GoogleMap map) {
 
         GPSTracker tracker = new GPSTracker(this);
         if (!tracker.canGetLocation()) {
-            tracker.showSettingsAlert();
+            displayLocationRequest(getApplicationContext());
```

Comment:
Can we move this function to `GPSTracker` file only? It might be required at other places also, we don't want to copy this function again.